plugins/container/go-worker/pkg/container/docker.go

```diff
@@ -139,9 +143,29 @@ func (dc *dockerEngine) ctrToInfo(ctx context.Context, ctr types.ContainerJSON)
 		}
 		containerPort := port.Int()
 		for _, portBinding := range portBindings {
+			rawHostIP, rawHostPort := portBinding.HostIP, portBinding.HostPort
+			// Parse IP address to uint32.
+			addr, err := netip.ParseAddr(rawHostIP)
+			if err != nil {
```

Comment:
Also, i'd avoid the error in case of malformed address, just skip it.
This should never happen and if it happens we still want to send all other metadata but this one IMHO.

Reply:
Yes! Better missing metadata fields than missing metadata altogether :)